### PR TITLE
[DO NOT MERGE] Have a common place for QE scripts used in multiple repositories (to avoid duplication)

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Have a common place for QE scripts used in multiple repositories (to avoid duplication)

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/16923

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
[ci-test-devfiles]

**Comment by Ohrimenko1988** [comment]
[ci-test-devfiles]

**Comment by benoitf** [comment]
@Ohrimenko1988 related issue is closed but we still have this PR opened with a user repo 

Can this PR be updated ?

File: .ci/cico-devfile-pr-test.sh [hunk 1]
```diff
@@ -11,91 +11,43 @@
...
+#Download and import the "common-qe" functions
 export IS_TESTS_FAILED="false"
-export FAIL_MESSAGE="Build failed."
-
-SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
-export SCRIPT_DIR
-
-. "${SCRIPT_DIR}"/che-util.sh
-. "${SCRIPT_DIR}"/installation-util.sh
+DOWNLOADER_URL=https://raw.githubusercontent.com/eclipse/che/iokhrime-common-centos/tests/.infra/centos-ci/common-qe/downloader.sh
```
Comment (benoitf) [inline .ci/cico-devfile-pr-test.sh:19]: should not be a user repo/branch

**Comment by ericwill** [comment]
We are using GitHub actions now, so this is no longer relevant.

**Pull request closed without merging**